test/common/upstream/BUILD

```diff
@@ -146,6 +146,44 @@ envoy_benchmark_test(
     benchmark_binary = "eds_speed_test",
 )
 
+envoy_cc_benchmark_binary(
+    name = "cds_speed_test",
+    srcs = ["cds_speed_test.cc"],
+    external_deps = [
+        "benchmark",
+    ],
+    deps = [
+        ":utility_lib",
+        "//source/common/config:grpc_mux_lib",
+        "//source/common/config:grpc_subscription_lib",
+        "//source/common/config:protobuf_link_hacks",
+        "//source/common/config:utility_lib",
+        "//source/common/upstream:cds_api_lib",
+        "//source/common/upstream:eds_lib",
+        "//source/extensions/transport_sockets/raw_buffer:config",
+        "//source/server:transport_socket_config_lib",
+        "//test/config:utility_lib",
+        "//test/mocks/local_info:local_info_mocks",
+        "//test/mocks/protobuf:protobuf_mocks",
+        "//test/mocks/runtime:runtime_mocks",
+        "//test/mocks/server:admin_mocks",
+        "//test/mocks/server:instance_mocks",
+        "//test/mocks/ssl:ssl_mocks",
+        "//test/mocks/upstream:cluster_manager_mocks",
+        "//test/test_common:test_runtime_lib",
+        "//test/test_common:utility_lib",
```

Comment:
Nit: might be worth a quick pass to IWYU and minify any deps that no longer apply. Generally we're not super strict, but this list is pretty long.

Reply:
I didn't manage to run IWYU successfully, but I did a pass and removed the obviously vestigial entries.